jupyter-book/tuning/parameter_tuning_module_take_away.md

```diff
@@ -9,10 +9,10 @@
 - The search for the best hyperparameters can be automated with a grid-search
   approach or a randomized search approach;
 - A grid-search is expensive and does not scale when the number of
-  hyperparameters to optimize increase. Besides, the combination are sampled
+  hyperparameters to optimize increase. Additionally, the combination are sampled
   only on a regular grid.
 - A randomized-search allows a search with a fixed budget even with an
-  increasing number of hyperparameters. Besides, the combination are sampled
+  increasing number of hyperparameters. Additionally, the combination are sampled
   on a non-regular grid.
```

Comment:
> - A randomized-search allows exploring within a fixed budget, even as the number
  of hyperparameters increases. In this case, combinations can be sampled either
  on a regular grid or from a given distribution.